sdk/nodejs/resource.ts

```diff
@@ -374,17 +384,18 @@ export class Output<T> {
         return obj && obj.__pulumiOutput;
     }
 
-    /* @internal */ public static create<T>(
-            resource: Resource, promise: Promise<T>, isKnown: Promise<boolean>): Output<T> {
-        return new Output<T>(new Set<Resource>([resource]), promise, isKnown);
-    }
```

Comment:
Not following the feedback entirely. I eliminated this and just made the (internal) constructor work for the set of cases that our internal code needs. And I fixed up the public `create` factories, so we can use those instead of the `pulumi.output` function which we agreed is a weird pattern we would move away from, and towards something more like `Promise.fulfilled`.